lib/teamtailor/client.rb

```diff
@@ -36,6 +36,22 @@ def candidates(page: 1, include: [])
       ).call
     end
 
+    def create_candidate(attributes:)
+      Teamtailor::Request.new(
+          base_url: base_url,
+          api_token: api_token,
+          api_version: api_version,
+          path: "/v1/candidates",
+          method: :post,
+          body: {
+              data: {
+                  type: "candidates",
+                  attributes: attributes.transform_keys { |k| k.to_s.gsub("_", "-") },
```

Comment:
This should also take `relationships`, right?

Reply:
Right!